src/annotations/selectors/index.ts

```diff
@@ -34,6 +34,13 @@ export const getHiddenAnnotationStreams = (
   return sortBy(filtered, stream => stream.name)
 }
 
+export const getAnnotationStreams = (
+  state: AppState
+): AnnotationStreamDetail[] => {
+  const annotationStreamDetails = state.annotations.streams
```

Comment:
style nit: i think what's happening here is clear enough to not to need this variable